ci/npm-publish.yml

```
@@ -27,7 +32,10 @@ jobs:
         with:
           node-version: 20
           registry-url: https://registry.npmjs.org/
+      # Ensure npm 11.5.1 or later is installed
+      - name: Update npm
+        run: npm install -g npm@latest
       - run: npm ci
+      # It is recommend to publish to npm using trusted publishing; for details,
+      # see: https://docs.npmjs.com/trusted-publishers
       - run: npm publish
```

We may want to leave the token commented out, as a fallback for GHES, and GHEC w/ data residency.

```suggestion
      - run: npm publish
      env:
          # An explicit token is needed only for non-github.com workflows.
          # NODE_AUTH_TOKEN: ${{secrets.npm_token}}
```

---

I think a bare `env:` will [cause an error](https://github.com/steiza/npm-provenance-test/actions/runs/18375381180), but I pushed up a riff on this suggestion.